charts/kubernetes-operations/alerts/kubernetes-node.yaml

```
@@ -153,3 +153,16 @@ groups:
       description: The node `{{`{{ $labels.node }}`}}` rebooted `{{`{{ $value | humanize }}`}}` times in the past hour. It could be stuck in a reboot/panic loop.
       summary: Node rebooted multiple times.
 {{- end }}
+
+{{- if not (.Values.prometheusRules.disabled.NodeInMaintenance | default false) }}
+  - alert: NodeInMaintenance
+    expr: {{ printf "max by (node) (kube_node_labels{label_%s='%s'}) == 1" .Values.prometheusRules.NodeInMaintenance.label .Values.prometheusRules.NodeInMaintenance.value }}`
+    for: 2m
+    labels:
+      severity: {{ dig "NodeInMaintenance" "severity" "none" .Values.prometheusRules }}
+      runbook_url: https://github.com/cloudoperators/kubernetes-operations/playbooks/NodeInMaintenance.md
```

That URL will always be a 404 but I could also not find the file in the repo at all.

---

This is just a placeholder. We need the experts here to write meaningful playbooks. 

Tracked in this issue: https://github.com/cloudoperators/kubernetes-operations/issues/2

---

it must be at least as that other route is never going to not show a 404
```suggestion
      runbook_url: https://github.com/cloudoperators/kubernetes-operations/blob/main/playbooks/NodeInMaintenance.md
```